src/pages/guides/using/contentlogs/contentlogs-setup.md

```diff
@@ -0,0 +1,179 @@
+            - Select a pre-made runtime action/runtime namespace
+- After Saving we need to add I/O Management API as a requirement to project
+- Select `Add API to Project`
+
+  ![Add API](../../img/contentlogs/add_api.png "Add I/O Management API to project")
+
+- Filter by `Adobe Services`
+
+  ![Filter API Selection](../../img/contentlogs/filter_adobe_services.png "Filter by Adobe Services")
+
+- Select `I/O Management API and click Next`
+
+  ![API selection](../../img/contentlogs/add_io_management_api.png "Select I/O Management API")
+
+- Select existing OAuth Server-to-Server Authentication and click `Save configured API`
+
+  ![Add OAuth](../../img/contentlogs/use_existing_oauth.png "Select existing Oauth")
+
+- Verify Streaming of Content Logs Working. Click Event Registration
+
+  ![Select Event Registration](../../img/contentlogs/go_to_event_registration.png "Select Event Registration")
+
+- Go to Event Browser
+
+  ![Select Event Browser](../../img/contentlogs/go_to_event_browser.png "Select Event Browser")
+
+- Click `Go to End`
+
+  ![Go To End Of Stream](../../img/contentlogs/go_to_end.png "Go to End")
+
+- Verify Response status 204
+
+  ![Verify Response 204](../../img/contentlogs/verify_204.png "Verify 204")
+
+- Open event registration in new tab
+
+  ![Go to Event Registration](../../img/contentlogs/go_to_event_registration.png "Go to event registration")
+
+- Click on send sample event say Updated
+
+  ![Send Sample Event](../../img/contentlogs/send_sample_event_read.png "Send Sample event")
+
+- Go to Event Browser Tab. Click on `Go To Next Batch`
+
+  ![Go To Next Batch](../../img/contentlogs/go_to_next_batch.png "Go to Next Batch")
+
+- Verify Response Status 200 with content log of the updated sample event
+
+  ![Verify Resopnse 200](../../img/contentlogs/verify_200.png "Verify 200")
+  ![Check Payload](../../img/contentlogs/sample_event_received.png "Sample Event received in stream")
+
+
+## Programmatic Setup for Consumption of Content Logs Stream
+
+Once the above setup is complete and validated, we can continuously consume the stream of content logs from Adobe I/O events as and when operations are performed on assets.
+
+- Use the exposed event registration endpoint for the project.
+
+  ![Journaling API Endpoint](../../img/contentlogs/journaling_api_endpoint.png "Endpoint for Content Logs stream consumption")
+
+- Auth token needs to be passed in the request. Which can be generated using the cURL and all credentials under Oauth of the project.
+
+  ![Auth Token Endpoint](../../img/contentlogs/auth_token_endpoint.png "Endpoint for generating AuthToken")
+
+- Sample cURL for consuming the events stream
+
+  ![cURL for stream consumption](../../img/contentlogs/cURL_stream_consumption.png "Sample cURL for consumption")
+
+- Once we invoke the API, response body has `_page.last` token which needs to be passed in subsequent calls to fetch more events.
```

Comment:
would prefer using the next link from headers as it contains the events-fast url